pkg/console/operator/operator.go

```diff
@@ -22,13 +22,15 @@ import (
 	oauthinformersv1 "github.com/openshift/client-go/oauth/informers/externalversions/oauth/v1"
 	routeclientv1 "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1"
 	"github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/vendor.bak/github.com/openshift/operator-boilerplate/openshift/operator"
```

Comment:
Is this iomport correct ? Looks wrong, but maybe I'm wrong 😄

Reply:
This is incorrect.  I will fix when that repo settles down.  Not sure if it will, or if we need to inline the files.  Discussion in progress.